partner_communication/report/a4_no_margin.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <report id="report_a4_no_margin"
+            model="partner.communication.job"
+            string="A4 BVR no margin    "
+            report_type="qweb-pdf"
+            name="report_compassion.a4_bvr_no_margin"
+    />
+    <record id="report_a4_no_margin" model="ir.actions.report.xml">
+        <field name="paperformat_id" ref="l10n_ch_payment_slip.paperformat_paymentslip"/>
+    </record>
+
+    <template id="a4_no_margin">
+        <t t-call="report_compassion.partner_communication"/>
```

Comment:
We should not use `ch` modules here! I think we can just remove this template because it's never used.